### PR TITLE
Bump to v4.1.2

### PR DESCRIPTION
**Pull request by timolxl**
Updated file paths and names, and added example for downloading the SDK with Curl.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +51,7 @@ cp -R "pesdk-html5-build-$VERSION/assets/"* vendor/assets/images
 
 4. Create new home controller with index page
 
-```bash 
+```ruby
```
Comment (maltebaumann) [inline README.md]: This was actually right, as the command is executed using bash (in the shell) and not written in Ruby. But should not really effect the syntax highlighting anyway.

File: README.md [hunk 2]
```diff
@@ -89,7 +99,7 @@ window.onload = ->
   return
 ```
 
-or create `app/assets/javascripts/home.js` and insert
+If you don't want to use CoffeeScript, delete `app/assets/javascripts/home.coffee`, create `app/assets/javascripts/home.js` and insert
 
 ```javascript 
 window.onload = function () {
```
Comment (maltebaumann) [inline README.md]: Can you update the `apiKey` variable and comment to match the explanation in the React demo? We need to make clear, that the SDK doesn't require API-Keys, but the contents of the license files.

File: README.md (hunk 1 above)
Comment (timolxl) [reply README.md]: So should I remove all syntax highlighting extensions?

Comment (maltebaumann) [reply README.md]: No, just change it back to bash :)

**Comment by maltebaumann** [comment]
Just discovered: You haven’t actually updated the demo rails app in the repository, just the README. The (probably old) SDK is included in vendor/assets.

**Pull request merged**